benchmarks/ddp/benchmark.py

```diff
@@ -0,0 +1,276 @@
+#!/usr/bin/env python3
+#
+# Measure distributed training iteration time.
+#
+# This program performs a sweep over a) a number of model architectures, and
+# b) an increasing number of processes. This produces a 1-GPU baseline,
+# an 8-GPU baseline (if applicable), as well as measurements for however
```

Comment:
nit: how many

Reply:
Intentional (see https://www.merriam-webster.com/dictionary/however for "however many").